e2e/tests/upgrades/upgrade_test.go

```diff
@@ -405,3 +551,29 @@ func getICAVersion(chainAVersion, chainBVersion string) string {
 	// explicitly set the version string because the host chain might not yet support incentivized channels.
 	return icatypes.NewDefaultMetadataString(ibctesting.FirstConnectionID, ibctesting.FirstConnectionID)
 }
+
+// Status queries the status of the client by clientID
+func (s *UpgradeTestSuite) Status(ctx context.Context, chain ibc.Chain, clientID string) (string, error) {
```

Comment:
maybe this can be renamed to `ClientStatus` to stay consistent with the `ClientState` function?